README.md

```diff
@@ -1,12 +1,9 @@
 
 # sbt-graphql [![Build Status](https://travis-ci.org/muuki88/sbt-graphql.svg?branch=master)](https://travis-ci.org/muuki88/sbt-graphql) [ ![Download](https://api.bintray.com/packages/sbt/sbt-plugin-releases/sbt-graphql/images/download.svg) ](https://bintray.com/sbt/sbt-plugin-releases/sbt-graphql/_latestVersion) 
 
-> This plugin is an experiment at this moment.
-> SBT 1.x only
+> This plugin is sbt 1.x only and experimental.
 
-SBT plugin to generate and validate graphql schemas written with Sangria.
-
-See also: https://github.com/mediative/sangria-codegen
```

Comment:
Nothing to see there, since the code generator was integrated and cleaned up in this plugin.